File: Challenges/Kidus/python/LargestWorld.py
def longestWord(sentence):
    """
    This function takes a string as input and returns the longest word in the string.
    """
    words = sentence.split()
    longest_word = max(words, key=len)
    return longest_word

# Test

print(longestWord("Hello, World!"))   # Output: "World"
print(longestWord("Python is fun!"))  # Output: "Python"
print(longestWord("The quick brown fox jumps over the lazy dog."))  # Output: "quick"

File: Challenges/Kidus/js/LargestWorld.js
function longestWord(sentence) {
    // Split the sentence into an array of words
    const words = sentence.split(" ");  
    let longest = "";  // Initialize an empty string to store the longest word

    // Loop through each word in the array
    for (let word of words) {
        // If the current word is longer than the stored longest word, update it
        if (word.length > longest.length) {
            longest = word;
        }
    }
    return longest;  // Return the longest word found
}

// Test Cases
console.log(longestWord("The quick brown fox jumps over the lazy dog"));  // Output: "jumps"
console.log(longestWord("JavaScript is awesome"));  // Output: "JavaScript"
